modules/provenance.nf

```diff
@@ -4,7 +4,7 @@ process collect_provenance {
 
   executor 'local'
 
-  publishDir "${params.outdir}/${params.run_name}/${params.pipeline_short_name}-v${params.pipeline_minor_version}/Provenance_files", pattern: "${sample_id}_*_provenance.yml", mode: 'copy'
+  publishDir "${params.outdir}/provenance_files", pattern: "${sample_id}_*_provenance.yml", mode: 'copy'
```

Comment:
Can we put the provenance files in `"${params.outdir}/${sample_id}"`? That's what we do for other pipelines:

https://github.com/BCCDC-PHL/routine-assembly/blob/7493965edd4a3edefd080f64459152a0287f5d84/modules/provenance.nf#L7